docs-src/config.toml

```diff
@@ -39,3 +39,6 @@ name = "<i class='fa fa-bookmark'></i> <label>Contact us</label>"
 identifier = "contactus"
 url = "https://www.cobaltspeech.com/contact"
 weight = 20
+
+[markup.goldmark.renderer]
+unsafe= true
```

Comment:
What does this mean? The word `unsafe` makes me worry, so I'm just checking that we feel "unsafe" is actually "safe enough".

Reply:
"unsafe" means "allow HTML tags to be rendered as HTML, I know what I'm doing" ;-)

Reply:
Otherwise, all the < and > get escaped and show up verbatim